src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var previousToggle = (newModel.pageIndex === 0);
+        var nextToggle = (newModel.pageIndex === newModel.pageCount - 1);
+        that.previous.toggleClass(disabledStyle, previousToggle);
+        that.next.toggleClass(disabledStyle, nextToggle);
+        that.previous.children('a').attr('aria-disabled', previousToggle);
+        that.next.children('a').attr('aria-disabled', nextToggle);
```

Comment:
I was using aria-attributes for the first time, so had no clue about that and really really sorry for the linting issue. This is really something I should have not done. Sorry for the amount of time I have made you invest into reviewing this.